static/app/routes.tsx

```diff
@@ -1375,12 +1375,10 @@ function buildRoutes() {
     <Fragment>
       <IndexRoute component={make(() => import('sentry/views/replays/list'))} />
       <Route
-        path="dead-clicks/"
-        component={make(() => import('sentry/views/replays/deadRageClick/deadClickList'))}
-      />
-      <Route
-        path="rage-clicks/"
-        component={make(() => import('sentry/views/replays/deadRageClick/rageClickList'))}
+        path="dead-rage-clicks/"
```

Comment:
we will need up update urls.py to match


While we're there, i think the path should be `selectors/` instead, so it's a bit more generic, like if we had any type of click appearing there... and so it doesn't use these terms which are more product-y imo